.github/workflows/verify-pr-details.yml

```diff
@@ -0,0 +1,29 @@
+name: Check PR title format
+
+on:
+  pull_request:
+    types:
+      # Everytime a PR is opened, reopened
+      - opened
+      - reopened
+      # Everytime a PR is edited (for example: changing the title)
+      - edited
+      # Everytime the PR head branch is updated (for example after pushing a new commit)
+      - synchronize
+
+jobs:
+  # Job that checks the title of the PR & makes sure it fits the semantic versioning format
+  check-pr-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
```

Comment:
Is there a difference between this custom script and [this](https://github.com/lumigo-io/opentelemetry-java-distro/blob/main/.github/workflows/push-actions.yml#L23)

Reply:
Good to know! switched to the action use suggested